src/miniscript/astelem.rs

```diff
@@ -25,6 +25,7 @@ use bitcoin::blockdata::{opcodes, script};
 use bitcoin::hashes::hex::FromHex;
 use bitcoin::hashes::{hash160, ripemd160, sha256, sha256d, Hash};
 
+use descriptor::PkTranslate;
```

Comment:
If we are using PkTranslate for underlying Miniscript and Terminal, maybe we should just move the trait to lib.rs from descirptor/mod.rs